broker/setup.py

```diff
@@ -25,16 +25,5 @@
     ],
     include_package_data=True,
     zip_safe=False,
-    setup_requires=[
-        'cffi>=1.4.1',
-    ],
-    install_requires=[
-        'netfilter>=0.6.2',
```

Comment:
I assume this means we can also remove netfilter from the installation instructions?

CI will still need these dependencies as it runs old versions. Could you add a comment [in this file](https://github.com/wlanslovenija/tunneldigger/blob/master/.travis.yml) saying that we are installing the dependencies of *all* versions, not just the latest?

Reply:
Or rather, the things we are installing are just for client and test suite, not server (as that runs in LXC)... but the modules we are loading are for all of them.

Reply:
Ah, there's also [this list](https://github.com/wlanslovenija/tunneldigger/blob/c50ef46d78d797750979ebf2f8ddc5aa993a02ae/tests/tunneldigger.py#L47).

Reply:
Added a new commit that explains the situation. Thanks :)